apps/jobs/views.py

```diff
@@ -539,3 +540,38 @@ def users(self, request):
         return response.create_response(
             serialized_company_data.data, status.HTTP_200_OK
         )
+
+
+class ContactUsViewSet(viewsets.ModelViewSet):
+    """Company object viewsets
+    API: /api/v/contact-us
+    Database: jobs_contactmessage
+    Functions:
+        1. take input from user end
+        2. create contact message
+        3. contact message stored in database
+    """
+
+    # queryset = ContactMessage.objects.all()
+    serializer_class = ContactUsSerializer
+
+    def get_queryset(self):
+        return ContactMessage.objects.all()
+
+    @action(detail=False, methods=["post"])
+    def create_contact_message(self, request):
+        full_name = request.data.get("full_name")
+        email = request.data.get("email")
+        message = request.data.get("message")
+
+        validation_error = validationClass.validate_fields(full_name, email, message)
```

Comment:
you are passing three different variables to the `validate_fields` static method, but it accepts only a dictionary format. This will throw an error. Also, the `validate_fields` method checks for the specific keys like age, experience, email, website etc. There's no such validation for `full_name`, `message`. You have to manually add those validations either in this function or create another static method in `validators.py` to handle scenarios where you have to check if the passed variable contains only str values or not.